pkg/boot/bzimage/bzimage_decompress.go

```diff
@@ -12,6 +12,7 @@ import (
 	"io"
 	"os/exec"
 
+	"github.com/therootcompany/xz/v/v1/xz"
```

Comment:
if we're going to depend on an xz library, let's not depend on a new one. we already have github.com/ulikunitz/xz included below?